front/app/modules/commercial/content_builder/admin/components/ContentBuilderToolbox/index.tsx

```diff
@@ -60,49 +62,49 @@ const ContentBuilderToolbox = ({
       </SectionTitle>
       <DraggableElement
         id="e2e-draggable-two-column"
-        component={<TwoColumn columnLayout="1-1" id="twoColumn" />}
+        component={<TwoColumn columnLayout="1-1" />}
         icon="layout-2column-1"
-        label={formatMessage(messages.twoColumn)}
+        label={formatMessage(TwoColumn.craft.custom.title)}
       />
       <DraggableElement
         id="e2e-draggable-three-column"
         component={<ThreeColumn />}
         icon="layout-3column"
-        label={formatMessage(messages.threeColumn)}
+        label={formatMessage(ThreeColumn.craft.custom.title)}
       />
       <DraggableElement
         id="e2e-draggable-white-space"
         component={<WhiteSpace size="small" />}
         icon="layout-white-space"
-        label={formatMessage(messages.whiteSpace)}
+        label={formatMessage(WhiteSpace.craft?.custom.title)}
```

Comment:
Minor: Are there cases where craft might not be available? If so maybe we could do a similar check for other areas in this file